src/styles/blocks/card.css

```diff
@@ -0,0 +1,105 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
```

Comment:
box-sizing and margin these attributes can be assigned to all elements on the page using * {
}

Reply:
Disagree. The product card will not be used on its own, but will be part of other content. In this case, using * we will spoil other content

Reply:
usually we use the same box sizing for all element so that we would not be confused by different size calculation
so using * for this is ok

Reply:
all reset margin, padding and box-sizing move to *{